ibis/backends/clickhouse/compiler.py

```diff
@@ -41,7 +41,19 @@ def _aggregate(self, funcname: str, *args, where):
         has_filter = where is not None
         func = self.f[funcname + "If" * has_filter]
         args += (where,) * has_filter
-        return func(*args)
+
+        return func(*args, dialect=self.dialect)
+
+    @staticmethod
+    def _minimize_spec(start, end, spec):
```

Comment:
Shouldn't this be a rewrite instead?

Reply:
Can you propose a rewrite rule? I'm not sure this can be done as a rewrite: it's altering the way specific ibis values are interpreted in sqlglot.

Reply:
Ahh, this changes the sqlglot expression. Then can you rename the method to `_minimize_window_spec` and add a short comment about what it does?